src/controllers/userController.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const userServices = require('../services/userServices');
+
+function getAllUsers(_, res) {
+  const allUsers = userServices.getAllUsers();
+
+  res.send(allUsers);
+}
+
+function getUserById(req, res) {
+  const { userId } = req.params;
+
+  if (!userId) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const user = userServices.getByUserId(userId);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.statusCode = 200;
```

Comment:
You should use 'res.status(200)' instead of 'res.statusCode = 200' for setting the status code.

Reply:
You do it in the expenseController, why do you do it in another way here?